src/commands/misc/owlbot.js

```diff
@@ -0,0 +1,52 @@
+const { Command, CommandError, SwitchbladeEmbed, Constants } = require('../../')
+
+module.exports = class Owlbot extends Command {
+  constructor (client) {
+    super({
+      name: 'owlbot',
+      aliases: ['owl'],
+      category: 'misc',
+      requirements: {
+        apis: [ 'owlbot' ]
+      },
+      parameters: [{
+        type: 'string',
+        full: true,
+        missingError: 'commands:owlbot.noWord'
+      }]
+    }, client)
+  }
+
+  async run ({ t, channel }, word) {
+    channel.startTyping()
+    try {
+      const { data } = await this.client.apis.owlbot.request(word)
+      const embed = new SwitchbladeEmbed()
+        .setTitle(data.word)
+        .addFields(
+          data.definitions.map(d =>
+            ({
+              name: d.type,
+              value: `${d.definition}${d.example ? `\n"_${d.example}_"` : ''}`
+            })
+          )
+        )
+        .setColor(Constants.OWLBOT_COLOR)
+        .setFooter(
+          t('commands:owlbot.footer'),
+          'https://media.owlbot.info/dictionary/images/owlbot.png.400x400_q85_box-0,0,287,288_crop_detail.png'
+        )
+      if (data.pronunciation) embed.setDescription(data.pronunciation)
```

Comment:
Add these two slashes, they're important for written pronunciation :)

![image](https://user-images.githubusercontent.com/25179120/94991323-c678d280-0558-11eb-8e9d-4cb6cf37f2cb.png)